pkg/server/container_create.go

```diff
@@ -635,8 +640,8 @@ func (c *criService) addOCIBindMounts(g *generate.Generator, mounts []*runtime.M
 				return err
 			}
 			options = append(options, "rslave")
-			if g.Spec().Linux.RootfsPropagation != "rshared" &&
-				g.Spec().Linux.RootfsPropagation != "rslave" {
+			if g.Config.Linux.RootfsPropagation != "rshared" &&
```

Comment:
can we add a comment here explaining and specifying where this change comes from..

Reply:
Sure, I'll update the commit description. It's because of the deprecation.

Reply:
nvm looks good